index.js

```diff
@@ -150,7 +155,7 @@ function filter(patterns, opts) {
 
     fp = utils.unixify(fp, opts);
     while (i < len) {
-      var fn = matcher(patterns[i++], opts);
+      var fn = patterns[i++];
       if (!fn(fp)) {
         res = false;
```

Comment:
Nevermind my comment about the backslash tests, just noticed you already did that. :+1: